service/src/main/java/tds/assessment/repositories/impl/AssessmentMapper.java

```diff
@@ -1,4 +1,4 @@
-/***************************************************************************************************
+/* **************************************************************************************************
```

Comment:
This shows as a warning in IntelliJ since this uses JavaDoc style but isn't a JavaDoc.  I can revert this and change my settings to ignore this if we don't want this change.